cairo/src/model.cairo

```diff
@@ -213,29 +213,28 @@ namespace model {
 
     // @notice A struct representing the header of an Eth block.
     struct BlockHeader {
-        base_fee_per_gas: felt,
-        blob_gas_used: felt,
-        bloom_len: felt,
-        bloom: felt*,
+        parent_hash: Uint256,
+        ommers_hash: Uint256,
         coinbase: felt,
-        difficulty: felt,
-        excess_blob_gas: felt,
-        extra_data_len: felt,
-        extra_data: felt*,
+        state_root: Uint256,
+        transactions_root: Uint256,
+        receipt_root: Uint256,
+        withdrawals_root: Uint256,
```

Comment:
Should be an Option no? as per https://github.com/alloy-rs/alloy/blob/8132d691762f69d4385050335978654c5c37510a/crates/consensus/src/header.rs#L49